be/src/service/point_query_executor.cpp

```diff
@@ -267,6 +266,10 @@ Status PointQueryExecutor::_lookup_row_key() {
     SCOPED_TIMER(&_profile_metrics.lookup_key_ns);
     // 2. lookup row location
     Status st;
+    if (_version >= 0) {
+        CHECK(config::is_cloud_mode()) << "Only cloud mode support snapshot read at present";
+        RETURN_IF_ERROR(std::dynamic_pointer_cast<CloudTablet>(_tablet)->sync_rowsets(_version));
```

Comment:
Does `_tablet->sync_rowsets` has side effect on _tablet to influnce other queries?

Reply:
no side effect